java-checks/src/main/java/org/sonar/java/checks/PublicStaticFieldShouldBeFinalCheck.java

```diff
@@ -66,7 +67,8 @@ public void visitClass(ClassTree tree) {
   private boolean isPublicStaticNotFinal(VariableTree tree) {
     boolean isPublic = false, isStatic = false, isFinal = false;
 
-    for (Modifier modifier : tree.modifiers().modifiers()) {
+    for (ModifierKeywordTree modifierKeywordTree : tree.modifiers().modifiers()) {
```

Comment:
False positive. One of the previous commit change the return type of `tree.modifiers().modifiers()` to `ModifierKeywordTree`